naloge/poslovnez.tex

```diff
@@ -110,7 +110,56 @@
 \pgfslika[poslovnez-resitev2]
 \end{slika}
     
+\newpage
+\begin{small}
+\begin{algorithmic}
+\Function{MAKSIMALNIDOBIČEK}{$G = (V, E), s \in V, L : E \to \R$}
+    \State $d \gets$ seznam slovarjev z vrednostjo $- \infty$ za vsako vozlišče $v \in V$
+    \State $\pred \gets$ seznam slovarjev z vrednostjo $\Null$
+    za vsako vozlišče $v \in V$
+    \State $d[0][s] \gets 0$
+    \State $i \gets 0$
+    \State $\text{\sl trenutna} \gets \{s\}$
+    \While{$\lnot \text{\sl trenutna}.\isEmpty()$}
+        \State $i \gets i+1$
+        \If{$i > |V|$} \hfill $\exists$ pozitiven cikel
+            \State \Return graf ima negativen cikel 
+        \EndIf
+        \State $\text{\sl{naslednja}} \gets$ prazna množica
+        \For{$u \in \text{\sl trenutna}$}
+            \State $d[i][u] \gets d[i-1][u]$
+            \State $\pred[i][u] \gets u$
+        \EndFor
+        \For{$uv \in E$}
```

Comment:
Glede na to, da imaš množico `trenutna`, bo zadostovalo, da greš čez vozlišča v njej in čez sosede vsakega od njih.